contrib/postgres_fdw/deparse.c

```diff
@@ -2354,13 +2416,35 @@ deparseExpr(Expr *node, deparse_expr_cxt *context)
 		case T_Aggref:
 			deparseAggref((Aggref *) node, context);
 			break;
+	    case T_RowExpr:
+			deparseRowExpr((RowExpr *) node, context);
+			break;
 		default:
 			elog(ERROR, "unsupported expression type for deparse: %d",
 				 (int) nodeTag(node));
 			break;
 	}
 }
 
+static void
+deparseRowExpr(RowExpr *node, deparse_expr_cxt *context)
+{
+	StringInfo	buf = context->buf;
+	bool		first;
+	ListCell   *lc;
+
+	appendStringInfoString(buf, "(");
```

Comment:
originally it read like `appendStringInfoString(buf, "ROW(");`. I read somewhere that `ROW` is "syntax noise" when there are more than one columns in the row and forgot to get it back to its original, generic form.